cmake/modules/FindMaya.cmake

```diff
@@ -205,11 +205,17 @@ foreach(MAYA_LIB
     Image
     Foundation
     IMFbase
-    tbb
     cg
     cgGL
-    clew)
+    clew
+)
+if (CMAKE_BUILD_TYPE MATCHES Debug)
+    list(APPEND MAYA_LIBS_TO_FIND tbb_debug)
```

Comment:
I compile in debug all the time and just happened to notice one day that this one wasn't being found correctly in debug. We aren't using it in MayaUsd, which is why it never caused a problem.